stripe.go

```diff
@@ -283,6 +296,8 @@ func (s *BackendConfiguration) NewRequest(method, path, key, contentType string,
 			}
 
 			req.Header.Add("Idempotency-Key", idempotency)
+		} else if strings.ToUpper(req.Method) == http.MethodPost || strings.ToUpper(req.Method) == http.MethodPut || strings.ToUpper(req.Method) == http.MethodDelete {
```

Comment:
Given that the use of `PUT` is somewhat speculative already, I wonder if we should just add `PATCH` in here as well for good measure.

Reply:
@brandur-stripe Add `PATCH` or replace `PUT` with `PATCH`?

Reply:
Add `PATCH`.